templates/bridgeserver2.yaml

```diff
@@ -80,6 +70,14 @@ Parameters:
     Type: String
     Description: Instance type to use for Elastic Beanstalk Instances
     Default: t2.small
+    AllowedValues:
+      - t2.nano
+      - t2.micro
+      - t2.small
+      - t2.medium
+      - t2.large
+      - t2.xlarge
+      - t2.2xlarge
```

Comment:
should change to newest class t3,t3a or t4g https://www.learnaws.org/2020/12/19/t3-t3a-t4g/